### DISCUSSION TITLE
Owncast - Self-hosted live video streaming server

### DISCUSSION THREAD

**Discussion by sacanods** [coollabsio/coolify#4336] [Service Template Requests]
github：https://github.com/owncast/owncast
Doc：https://owncast.online/quickstart/container/
>Owncast is a free and open source live video and web chat server for use with existing popular broadcasting software.

Simply put, this is a simple live broadcast server. Able to quickly set up a live broadcast room.
I've tried using docker compose to create in Coolify. But I always get the "Bad gateway" error.

**Comment by basaracar** [comment]
You need to configure port 8080 to be enabled in the Network -> Ports section. It worked for me when I configured it that way.

**Reply by djsisson** [reply]
its due to owncast exposing two ports, so unless the web port is specified as the loadbalancer port, traefik defaults to the lowest port number, which in this case is not the web ui.

adding SERVICE_OWNCAST_8080 to the environment section, would tell coolify to use that port for the web ui
